### PR TITLE
Format code using 'cargo fmt'

### PR DESCRIPTION
**Pull request by Atul9**

### CODE REVIEW

File: src/cargo.rs [hunk 1]
```diff
@@ -1,15 +1,15 @@
+use rand::distributions::{ChiSquared, Distribution, Exp};
 /// Module that pretends to run cargo to install rust packages.
```
Comment (svenstaro) [inline src/cargo.rs]: I suppose this should be a `//!` right?

File: src/composer.rs [hunk 2]
```diff
@@ -1,11 +1,11 @@
+use rand::distributions::{ChiSquared, Distribution};
 /// Module that pretends to install composer packages.
```
Comment (svenstaro) [inline src/composer.rs]: This and all other instances should likely also be the module doc strings `//!` instead of `///`, probably.

**Comment by svenstaro** [comment]
Thanks, this seems useful. I'd like some minimal changes. I think the docstrings should be module docstrings actually.

File: src/cargo.rs (hunk 1 above)
Comment (Atul9) [reply src/cargo.rs]: there was no update made to this line `/// Module that pretends to run cargo to install rust packages.` by `cargo fmt`

Comment (svenstaro) [reply src/cargo.rs]: Well, it was re-arranged.

Comment (Atul9) [reply src/cargo.rs]: I will check if there is any config related to `module docstrings` in rustfmt.toml and reply by this weekend

Comment (Atul9) [reply src/cargo.rs]: I think `///` is correct. I tried updating to `//!` and ran `cargo check` I got - 
```error: expected outer doc comment
 --> src/cargo.rs:2:1
  |
2 | //! Module that pretends to run cargo to install rust packages.
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: inner doc comments like this (starting with `//!` or `/*!`) can only appear before items

error: aborting due to previous error

error: could not compile `genact`.
```

**Comment by svenstaro** [comment]
Code has been formatted.

**Pull request closed without merging**